cephalopod/webinterface/frontend.py

```diff
@@ -31,29 +29,23 @@ def get_server_fields(server):
 
 @bp.route('/')
 @menu('index')
+@login_required
```

Comment:
I'd go for this:

    cephalopod/webinterface/frontend.py - blueprint 'frontend'
    cephalopod/webinterface/auth.py - blueprint 'auth'